code/modules/mob/hear_say.dm

```diff
@@ -243,3 +250,9 @@
 		heard = "<span class = 'game_say'>...<i>You almost hear someone talking</i>...</span>"
 
 	src << heard
+
+/mob/proc/hear_dying(var/message, var/mob/M)
+	if(get_dist(src, M) <= 2)
```

Comment:
Maybe radio messages are just like "You hear a hiss in your ear..." or something